src/main/java/hw_jdi/entities/MetalsAndColorsDTO.java

```diff
@@ -0,0 +1,28 @@
+package hw_jdi.entities;
+
+import hw_jdi.enums.*;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Setter
+@Getter
+public class MetalsAndColorsDTO {
+    public List<Integer> summary;
+    public List<String> elements;
+    public String color;
+    public String metals;
+    public List<String> vegetables;
+
+    public static MetalsAndColorsData getMetalAndColorsFromDTO(MetalsAndColorsDTO dto) {
```

Comment:
Could you please tell me, what is the reason of using DTO in this particular work ?